general-configuration-guide/use-tinymce-distraction-free.md

```diff
@@ -6,80 +6,32 @@ description: Mode that renders a lightweight UI for inline editing.
 keywords: Mode inlite distraction-free
 ---
 
-The **Distraction-free** mode renders the editor in lightweight inline mode. This mode provides options to quickly insert links, images, and tables into the content.
-
-To try out a {{site.productname}} editor in distraction-free mode, see the [Distraction-free editor]({{ site.baseurl }}/demo/editor-dfree/) example page.
+The **Distraction-free** mode renders the editor in inline mode without the menu bar or toolbar. Most of the editor functionality is accessed using: keyboard shortcuts, contextual menus, contextual toolbars, and toolbars provided by the `quickbars` plugin.
```

Comment:
Quickbars are just context toolbars, so I feel as though this could mislead customers later?

Reply:
Now
>  and toolbar functions provided by the `quickbars` plugin.